debugtools/DDR_Autoblob/src/com/ibm/j9ddr/autoblob/GenerateBlobC.java

```diff
@@ -211,7 +212,7 @@ private static void writeCFile(File inputFile, StructureXMLParser parser, Genera
 		}
 	}
 	
-	private static void loadTypes(SortedSet<UserDefinedType> typesToWrite, Collection<? extends UserDefinedType> toLoad, GenerateBlobCConfig config)
```

Comment:
Why was private removed?

Reply:
Removing the `private` modifier avoids the need for a bridge method.